### PR TITLE
Fix case sensitivity for 'click' package

### PR DESCRIPTION
**Pull request by formulahendry**
'click' package should be lower case: https://pypi.org/project/click/

### CODE REVIEW

**Comment by jongio** [comment]
Just curious was this breaking something?

**Comment by formulahendry** [comment]
@jongio Yes

Environment:

- Python 3.6
- Windows 10

Repro steps:

1. pip install iotedgedev
2. pip list (you could see `click` (lower case) is installed)
3. set up Jenkins on the same machine
4. try to run `iotedgedev ***` in Jenkins pipeline
5. You will see error message like below

```
[TestEdge] $ cmd /c iotedgedev genconfig
Traceback (most recent call last):
  File "c:\program files (x86)\python36-32\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\program files (x86)\python36-32\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Program Files (x86)\Python36-32\Scripts\iotedgedev.exe\__main__.py", line 5, in <module>
  File "c:\program files (x86)\python36-32\lib\site-packages\iotedgedev\cli.py", line 9, in <module>
    import click
ModuleNotFoundError: No module named 'click'
```

**Comment by jongio** [comment]
Okay, looks like good to merge once the ci is fixed.

**Comment by blackchoey** [comment]
@formulahendry please help verify whether this fixes the Jenkins error after merged.

File: requirements.txt [hunk 1]
```diff
@@ -15,4 +15,4 @@ six
 applicationinsights
 commentjson
 pypiwin32==219; sys_platform == 'win32' and python_version < '3.6'
-pypiwin32==223; sys_platform == 'win32' and python_version >= '3.6'
\ No newline at end of file
+pypiwin32==223; sys_platform == 'win32' and python_version >= '3.6'
```
Comment (blackchoey) [inline requirements.txt]: Seems the strings are identical, what's changed?

Comment (formulahendry) [reply requirements.txt]: I just change `click` through GitHub web portal. I guess GitHub has do some adjustment for the line ending which should  not have side effect.

**Pull request merged**